From 61891b5a95a97858e5f36f42f95b579817344994 Mon Sep 17 00:00:00 2001
From: bb10 <607398+bb10@users.noreply.github.com>
Date: Wed, 16 Jan 2019 06:13:38 +0100
Subject: [PATCH] Update joystick.xml

Added <back>ActivateWindow(Home)</back> as a global keypress, it was not assigned any action before.
Changed <b> back to the original action of Stop and changed the long press action to Fullscreen.
Changed the <back> button to the Fullscreen action. It doesn't toggle, instead it sends you back to the home screen after pressing it a second time because of the global action, which kind of makes sense for the View (on xbox one, previously back) button.
---
 system/keymaps/joystick.xml | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/system/keymaps/joystick.xml b/system/keymaps/joystick.xml
index c4b50cd4829b..9edf770ecbcd 100644
--- a/system/keymaps/joystick.xml
+++ b/system/keymaps/joystick.xml
@@ -49,6 +49,7 @@
       <x>ContextMenu</x>
       <y>FullScreen</y>
       <start>ActivateWindow(PlayerControls)</start>
+      <back>ActivateWindow(Home)</back>
       <guide>ActivateWindow(Home)</guide>
       <up>Up</up>
       <down>Down</down>
@@ -91,12 +92,12 @@
   <FullscreenVideo>
     <joystick profile="game.controller.default">
       <a>Pause</a>
-      <b>Fullscreen</b>
-      <b holdtime="500">Stop</b>
+      <b>Stop</b>
+      <b holdtime="500">FullScreen</b>
       <x>OSD</x>
       <y>FullScreen</y>
       <start>Info</start>
-      <back>OSD</back>
+      <back>FullScreen</back>
       <guide>OSD</guide>
       <up>ChapterOrBigStepForward</up>
       <down>ChapterOrBigStepBack</down>
